### DISCUSSION TITLE
[OP Stack Deployment] Deploy Sepolia

### DISCUSSION THREAD

**Discussion by avenbreaks** [ethereum-optimism/developers#18] [Deploying the OP Stack] [answered]
### Issue Description

GN Guys, we deploying OP Stack using sepolia and use RPC from quicknode but we getting error. and op-node script we run : 

Script : 

```
#!/bin/bash

nohup ./bin/op-node \
	--l2=ws://localhost:8551 \
	--l2.jwt-secret=./jwt.txt \
	--sequencer.enabled \
	--sequencer.l1-confs=5 \
	--verifier.l1-confs=4 \
	--rollup.config=./rollup.json \
	--rpc.addr=0.0.0.0 \
	--rpc.port=8548 \
	--p2p.disable \
	--rpc.enable-admin \
	--p2p.sequencer.key=$SEQ_KEY \
	--l1=$$L1_RPC \
	--l1.rpckind=$RPC_KIND \
	> /home/ubuntu/op-geth/logs/op-node.log &
```

RPC_KIND=quicknode
L1_RPC=quicknode-rpc-api
SEQ_KEY=privkeywithout0x

Logs : 

```
t=2023-10-26T14:22:39+0000 lvl=info msg="Initializing rollup node"            version=v0.0.0-b4201bdf-1698299710
t=2023-10-26T14:23:45+0000 lvl=eror msg="Error initializing the rollup node"  err="failed to init L1: failed to get L1 RPC client: failed to dial L1 address (150544L1_RPC): operation failed permanently after 10 attempts: failed to dial address (150544L1_RPC): dial unix 150544L1_RPC: connect: no such file or directory"
t=2023-10-26T14:23:45+0000 lvl=crit msg="Application failed"                  message="failed to setup: unable to create the rollup node: failed to init L1: failed to get L1 RPC client: failed to dial L1 address (150544L1_RPC): operation failed permanently after 10 attempts: failed to dial address (150544L1_RPC): dial unix 150544L1_RPC: connect: no such file or directory"
```

**Comment by sbvegan** [comment, marked as answer]
It looks like you have two $ in your environment variable: `--l1=$$L1_RPC`, can you try removing one and see if that works?

**Reply by avenbreaks** [reply]
ah sorry i can't see but now i got this : 

```
t=2023-10-26T14:34:53+0000 lvl=eror msg="failed to fetch runtime config data" err="failed to fetch unsafe block signing address from system config: failed to fetch proof of storage slot 0x65a7ed542fb37fe237fdfbdd70b31598523fe5b32879e307bae27a0bd9581c08 at block 0x944bbf0dedad036ad781108dd90d1db82f5d63e9b9f885bf62a621ca2cfcdad9: Invalid string length"
t=2023-10-26T14:35:03+0000 lvl=eror msg="failed to fetch runtime config data" err="failed to fetch unsafe block signing address from system config: failed to fetch proof of storage slot 0x65a7ed542fb37fe237fdfbdd70b31598523fe5b32879e307bae27a0bd9581c08 at block 0x846d66c9428c83fca71d824541bbdb0a9a2e151b753256ff6618bae972ecda1b: Invalid string length"
```

**Reply by avenbreaks** [reply]
i use anvil to fork sepolia node from quicknode

**Reply by opfocus** [reply]
This is the key, op-stack native code does not support anvil

**Reply by avenbreaks** [reply]
how i can get more testnet faucet from L1 ?

**Reply by sbvegan** [reply]
Our [faucet](https://app.optimism.io/faucet) will support Sepolia soon and we'll be updating it soon to support bulk requests. In the meantime, can you share your testnet wallet address and I can get you started with the suggested amount?

**Reply by avenbreaks** [reply]
i wish got bulk request sepolia faucet, please notice on here sir.

You can suggest to us how much is enough to build a testnet. because the documentation only suggests enough of 4 addresses. 

Address : 
0x00000023DfCdE07675ac95bd09f5DBC2814fEb90
0x111111C06CB458462815987F88C95F6A3729cf53

**Reply by opfocus** [reply]
please open a new thread

**Reply by avenbreaks** [reply]
yes sir, this tread closed. waiting faucet bulk.

**Reply by sbvegan** [reply]
@avenbreaks Our [documentation](https://stack.optimism.io/docs/build/getting-started/#generate-some-keys) currently recommends 17 ETH
> Recommended funding amounts are as follows:
> Admin — 2 ETH
> Proposer — 5 ETH
> Batcher — 10 ETH

I've sent funds to the first address (That's wild your addresses have so many leading 0s and 1s)

**Reply by avenbreaks** [reply]
thankyou sir

**Reply by avenbreaks** [reply]
ah, generate from geth and using vanityeth

**Comment by avenbreaks** [comment]
> Our [faucet](https://app.optimism.io/faucet) will support Sepolia soon and we'll be updating it soon to support bulk requests. In the meantime, can you share your testnet wallet address and I can get you started with the suggested amount?



> please open a new thread
